styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+@media only screen and (max-width: 600px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    width: 100vw;
+    height: 3rem;
+    padding: 1rem;
+  }
+
+  .navbar .shortcuts {
+    display: none;
+  }
+
+  .options {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+
+  .social i {
+    font-size: 2rem;
+    color: rgb(59, 59, 59);
+  }
+
+  .options i {
+    font-size: 1.5rem;
+  }
+
+  .social i:hover {
+    color: rgb(45, 204, 151);
+  }
+
+  .options i:hover {
+    color: rgb(45, 204, 151);
+  }
+
+  #homepage {
+    display: flex;
+    max-width: 100vw;
+    min-height: 100vh;
+    background-image: url(header-illsutration-mobile@2x.png);
+    background-size: 100vw;
+    border-bottom-left-radius: 2rem;
+    flex-direction: column;
+    padding: 2rem;
+    justify-content: space-around;
+  }
+
+  #homepage h1 {
+    color: rgb(0, 0, 88);
+    font-size: 2.5rem;
+    text-align: center;
+  }
+
+  #homepage h1 span {
+    color: rgb(45, 204, 151);
+  }
+
+  #homepage p {
+    font-size: larger;
```

Comment:
Kindly try to follow Figma design, font, styles, etc. as much as you can.
- As per Figma design, this `font-size` is `20px`
- Also add `line-height` to the paragraph, which is missing here.